modules/common/CHANGELOG.md

```diff
@@ -17,8 +17,18 @@ TEMPLATE:
 -->
 
 ## [UPCOMING]
+### Adds
+- Adds `isProposalId()` function to the package
+
+### Changed
+- Changes `encodeProposalId` and `decodeProposalId` function to support the new format
+
+## 0.11.0-beta
+- Adds `encodeProposalId(addr, nonce)` and `decodeProposalId(pid)`
```

Comment:
Move up, where `UPCOMING` is now. Versions are no longer automatic, right?

Reply:
moved